Gordon360/Services/HousingService.cs

```diff
@@ -20,6 +20,75 @@ public HousingService(IUnitOfWork unitOfWork)
             _unitOfWork = unitOfWork;
         }
 
+        /// <summary>
+        /// Calls a stored procedure that returns a row in the staff whitelist which has the given user id,
+        /// if it is in the whitelist
+        /// </summary>
+        /// <param name="userId"> The id of the person using the page </param>
+        /// <returns> Whether or not the user is on the staff whitelist </returns>
+        public bool CheckIfHousingAdmin(string userId)
+        {
+            IEnumerable<ApartmentAppAdminViewModel> idResult = null;
+
+            var idParam = new SqlParameter("@USER_ID", userId);
+
+            idResult = RawSqlQuery<ApartmentAppAdminViewModel>.query("GET_AA_ADMIN @USER_ID", idParam);
+            if (!idResult.Any())
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        /// <summary>
+        /// Calls a stored procedure that inserts the given id into the whitelist table
+        /// </summary>
+        /// <param name="id"> The id to insert </param>
+        /// <returns> Whether or not this was successful </returns>
+        public bool AddHousingAdmin(string id)
+        {
+            IEnumerable<ApartmentAppAdminViewModel> idResult = null;
+
+            var idParam = new SqlParameter("@ADMIN_ID", id);
+
+            idResult = RawSqlQuery<ApartmentAppAdminViewModel>.query("INSERT_AA_ADMIN @ADMIN_ID", idParam);
+            if (idResult == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The id could not be added." };
```

Comment:
Not sure if a `ResourceNotFoundException` is the right error for the 'add' case, but it will at least have the intended result of giving an error

Reply:
Oh, true. I could fix that pretty quick, I think

Reply:
Oh, this is actually what other Post endpoints too. It should still be looked into, but maybe later as a larger refactoring.

Reply:
Agreed